packages/xrpl/src/index.ts

```diff
@@ -7,6 +7,8 @@ export * from './models'
 
 export * from './utils'
 
+export * from './sugar'
```

Comment:
The `sugar` folder shouldn't be exported directly, it should only be exported via methods on the client.